modules/apps/object/object-rest-api/src/main/java/com/liferay/object/rest/dto/v1_0/ObjectEntry.java

```diff
@@ -397,7 +397,33 @@ public void setKeywords(
 	@JsonAnyGetter
 	@Schema
 	@Valid
+	public Map<String, UnsafeSupplier<Object, Exception>> getLazyProperties() {
+		return _lazyProperties;
+	}
+
 	public Map<String, Object> getProperties() {
+		if (properties != null) {
+			return properties;
+		}
+
+		if (_lazyProperties != null) {
+			properties = new HashMap<>();
```

Comment:
I am worried about two processes in parallel calling the `getProperties` method. I think it could return different information. For example, the first one would create a map and before inserting all the elements from the _lazyProperties, another process would call and would get an empty map. I think we need to ensure that is not possible